### PR TITLE
Replaces existing font with BradMicks Custom Apache Font

### PR DESCRIPTION
**Pull request by BradMick**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp [hunk 1]
```diff
@@ -556,7 +563,7 @@ class Mpd_Arrow_##name##_Triangle { \
                 { {startX + 0.4*MPD_TEXT_HEIGHT, startY - 0.25*MPD_TEXT_HEIGHT}, 1 },\
                 { {startX + 0.4*MPD_TEXT_HEIGHT, startY + 1.25*MPD_TEXT_HEIGHT}, 1 },\
                 { {startX + 0.2*MPD_TEXT_HEIGHT, startY + 1.25*MPD_TEXT_HEIGHT}, 1 },\
-                { {startX + 0.2*MPD_TEXT_HEIGHT, startY - 0.25*MPD_TEXT_HEIGHT}, 1 }\
+                { {startX + 0.2*MPD_TEXT_HEIGHT, startY - 0.25*MPD_TEXT_HEIGHT}, 1 },\
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp:566]: Why was this comma added?

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp [hunk 2]
```diff
@@ -403,7 +410,7 @@ class Mpd_Arrow_##name##_Triangle { \
     and can fit an arrow within.
 
 Parameters:
-    name - Unique name to use in the class definition. Can be the same as that in <MPD_TEXT_x> as they are
+    name - Unique name to use in the class definition. Can be the same as that in <MPD_TEXT_x> as they areMPD_TEXT_HEIGHT
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp:413]: Was this change intended?

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp [hunk 3]
```diff
@@ -353,7 +360,7 @@
 #define MPD_ARROW_R(name, startX, startY, numChars) class Mpd_Arrow_##name##_Line { \
     type = line; \
     points[] = { \
-        {{startX, startY - MPD_ARROW_PAD}, 1}, \
+        {{startX + MPD_ARROW_LENGTH, startY - MPD_ARROW_PAD}, 1}, \
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp:363]: Why was this changed?

**Review by mattysmith22 — COMMENTED** [review]
Looks good overall! Just some quick questions about some changes made

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp (hunk 1 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp:566]: Not sure, deleted.

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp (hunk 2 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp:413]: Likely copy-pasta error, deleted

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp (hunk 3 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp:363]: This was meant to fix an issue where the arrow wasn't starting at the exact left edge IIRC.

Comment (mattysmith22) [reply @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp:363]: This seems to move the left point too far over on my computer? The tip of the arrow is in-line, but the left is pushed in

Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mpd_defines.hpp:363]: Reverted, seems to be working as intended.

**Review by mattysmith22 — APPROVED** [review]
Looks good!
